backend/src/services/user_service.py

```diff
@@ -1,9 +1,16 @@
 
 import requests
+from tornado.options import options
 from src.cache import cache
 from src.services.token_cache_service import TokenCacheService
 
-TOKEN_CHECK_URL = "https://account.chalmers.it/userInfo.php?token=%s"
+def get_lan_user(token):
+    return requests.get("http://lan.chalmers.it/?p=userInfo",headers={'Cookie': 'chalmers_lan_auth=%s' % token})
```

Comment:
Depending on how the config_provider works, shouldn't this cookie-name-thingy come from that?